src/doc/DocAnnotator.js

```diff
@@ -167,16 +167,16 @@ class DocAnnotator extends Annotator {
                 clientEvent.clientX - pageDimensions.left,
                 clientEvent.clientY - pageDimensions.top - PAGE_PADDING_TOP
             ];
-            let [x, y] = browserCoordinates;
 
             // Do not create annotation if event doesn't have coordinates
```

Comment:
Combine comments
`// Do not create annotation if event doesn't have coordinates or occurs outside the page`